utils/src/main/java/org/apache/cloudstack/utils/security/SSLUtils.java

```diff
@@ -49,10 +66,10 @@
     }
 
     public static SSLContext getSSLContext() throws NoSuchAlgorithmException {
-        return SSLContext.getInstance("TLSv1");
+        return SSLContext.getInstance("TLSv1.2");
     }
 
     public static SSLContext getSSLContext(String provider) throws NoSuchAlgorithmException, NoSuchProviderException {
-        return SSLContext.getInstance("TLSv1", provider);
+        return SSLContext.getInstance("TLSv1.2", provider);
```

Comment:
This might break clients (non-browser) for some cases where TLSv1.2 is not supported.